go-apps/meep-loc-serv/server/loc-serv.go

```diff
@@ -3810,3 +3834,67 @@ func distanceGet(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 	fmt.Fprintf(w, string(jsonResponse))
 }
+
+func mec011AppTerminationPost(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
+
+	var notification AppTerminationNotification
+	decoder := json.NewDecoder(r.Body)
+	err := decoder.Decode(&notification)
+	if err != nil {
+		log.Error(err.Error())
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	if !appEnablementEnabled {
+		//just ignore the message
+		w.WriteHeader(http.StatusNoContent)
+		return
+	}
+
+	stopRegistrationTicker()
```

Comment:
Is this call necessary? As soon as we are registered we stop this timer.

Reply:
overthinking: if registration was not complete... ticker is still running... but then again... service will not receive an appTermination.. so removing this code